src/main/java/cloud/fogbow/ras/core/processors/SpawningProcessor.java

```diff
@@ -70,6 +70,9 @@ private void processInstance(Order order) throws FogbowException {
         LocalCloudConnector localCloudConnector = (LocalCloudConnector) CloudConnectorFactory.getInstance().
```

Comment:
We don't need this casting, because it is guaranteed that the CloudConnector is a local one. Instead of this command, use:
LocalCloudConnector localCloudConnector = new LocalCloudConnector(order.getCloudName());